server/service/client.go

```diff
@@ -559,6 +559,65 @@ func (c *Client) ApplyGroup(
 			tmScriptsPayloads[k] = scriptPayloads
 		}
 
+		tmSoftware := extractTmSpecsSoftware(specs.Teams)
```

Comment:
Just for my understanding, this adds the support for both `fleetctl apply` and `fleetctl gitops`? They both end up calling this `ApplyGroup`?

Reply:
Argh :( I thought this was good for both. Let coordinate in slack if there are ways to split work for all the remaining bits.

Reply:
sounds good, thank you!!